app/views/components/fields/field_text_multiple_values.scala.html

```diff
@@ -24,7 +24,8 @@
     elements: Seq[String] = Seq.empty,
     inputClass: Option[String] = None,
     labelClass: Option[String] = None,
-    includeAddButton: Boolean = true
+    includeAddButton: Boolean = true,
+    valuePrefix: Option[String] = None
```

Comment:
What's `valuePrefix` for?

Reply:
Differentiating the ids of different types of list items on the same form.  (Only used in Item Type page at present)